src/App.tsx

```diff
@@ -15,37 +15,90 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+const App: React.FC = () => {
+  const [list, setList] = useState<string[]>(goodsFromServer);
+  const [reverseList, setReverseList] = useState<boolean>(false);
+  const [sortedState, setSortedState] = useState<number>(0);
+
+  const reset = () => {
+    setList(goodsFromServer);
+    setReverseList(false);
+    setSortedState(0);
+  };
+
+  const sortedGoods = (goods: string[]) => {
+    setSortedState(1);
```

Comment:
Eduardo, I believe you missed one point of the [Checklist](https://github.com/mate-academy/react_list-of-goods-hooks#:~:text=Use%20enum%20SortType%20instead%20of%20string%20literals%20for%20possible%20sorting%20options%20(including%20the%20default%20one)) 

You should use an Enum to test the sorting types instead of numbers